resilience4j-framework-common/build.gradle

```diff
@@ -7,4 +7,4 @@ dependencies {
     compile project(':resilience4j-timelimiter')
 }
 
-ext.moduleName = 'io.github.resilience4j.framework-common'
\ No newline at end of file
+ext.moduleName = 'io.github.resilience4j.framework.common'
```

Comment:
Could you please change it into `io.github.resilience4j.frameworkcommon`

In the modules `spring-boot2` and `spring-boot` we also just removed `-`.